doc/qbk/06_websocket/02_handshaking.qbk

```diff
@@ -150,6 +150,19 @@ stream:
 
 [code_websocket_2_5]
 
+[heading Sub Protocols]
+
+The Websocket protocol understands the concept of sub-protocols. If the client
+is requesting one of a set of sub-protocols it will set the header `Sec-Websocket-Protocol`
+in the initial HTTP request. It is up to the server to parse the header and select one of the
+protocols to accept. The server indicates the selected protocol by setting the `Sec-Websocket-Protocol`
+header in the accept header.
+
+This is accomplished by means of a [link beast.ref.boost__beast__websocket__stream_base__decorator `decorator`]:
```

Comment:
You need to explain what the code does. For example:

"The code that follows demonstrates how a server reads an HTTP request, identifies it as a WebSocket Upgrade, and then checks for a matching subprotocol before performing the WebSocket handshake."